lib/registries/addon/overview/route.ts

```diff
@@ -1,12 +1,59 @@
 import { action } from '@ember-decorators/object';
 import { service } from '@ember-decorators/service';
+import HeadTagsService from 'ember-cli-meta-tags/services/head-tags';
+import { task } from 'ember-concurrency';
+import config from 'ember-get-config';
+import moment from 'moment';
 
+import Contributor from 'ember-osf-web/models/contributor';
+import Institution from 'ember-osf-web/models/institution';
 import Registration from 'ember-osf-web/models/registration';
 import GuidRoute, { GuidRouteModel } from 'ember-osf-web/resolve-guid/guid-route';
 import Analytics from 'ember-osf-web/services/analytics';
+import MetaTags, { HeadTagDef } from 'ember-osf-web/services/meta-tags';
+import Ready from 'ember-osf-web/services/ready';
+import pathJoin from 'ember-osf-web/utils/path-join';
 
 export default class Overview extends GuidRoute {
     @service analytics!: Analytics;
+    @service('head-tags') headTagsService!: HeadTagsService;
+    @service metaTags!: MetaTags;
+    @service ready!: Ready;
```

Comment:
Looks like `ready` isn't used -- call `ready.getBlocker()` before the first `yield` in `setHeadTags`